utils/vllm_build/roles/vllm_build/tasks/vllm_source_build.yml

```diff
@@ -31,9 +31,13 @@
 - name: Build vLLM latest container
   environment:
     BUILDKIT_HOST: "nerdctl-container://buildkitd"
-  ansible.builtin.command: nerdctl build -f "{{ vllm_dir }}/Dockerfile.rocm" -t vllm-rocm:latest "{{ vllm_dir }}"
+    http_proxy: "{{ proxy[0].http_proxy | default('', true) }}"
+    https_proxy: "{{ proxy[0].https_proxy | default('', true) }}"
+    HTTP_PROXY: "{{ proxy[0].http_proxy | default('', true) }}"
+    HTTPS_PROXY: "{{ proxy[0].https_proxy | default('', true) }}"
```

Comment:
NO_PROXY is missing

Reply:
this task does not require NO_PROXY variable as not accessing any local network